C/manual/ch_Finance-Quote.xml

```diff
@@ -278,7 +273,8 @@ SYNOPSIS
     <title>Using &app-cli; for Testing and Automation.</title>
 
     <abstract>
-      <para>&app; provides a commandline facility &app-cli; that one can use in a terminal session to check the
+      <para>&app; provides a commandline facility &app-cli;, which can be used in mode
+        <replaceable>--quotes</replaceable> that one can use in a terminal session to check the
```

Comment:
Terrible grammar. "which" leads a parenthetic that must be wrapped in commas, as in
```
&app-cli;, which can be used in mode <replaceable>--quotes</replaceable>, …
```
It doesn't work to interpose it with a "that"-phrase applying to the same object. Use
```
&app-cli; that one can use with the <replaceable>--quotes</replaceable> option in a terminal session to check the
```